templates/shared/forms/interactive/automotive.html

```diff
@@ -0,0 +1,81 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <h2 class="p-modal__title u-sv1" id="modal-title">Contact us</h2>
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <div class="u-sv2 js-formfield">
+        <h3 class="p-heading--4">What can we help you with?</h3>
+        <textarea id="open-question" name="open-question" aria-label="What can we help you with?" rows="3"></textarea>
```

Comment:
Similar to the conversations we had yesterday about the Hannover form, we can only pass specific inputs to Marketo, so `open-question` needs to be updated to `Comments_from_lead__c`, because currently the value of the textarea isn't being sent in the payload to Marketo. This screenshot shows what was sent, and I had entered "test test test" into the textarea:

![Screenshot from 2022-05-26 14-08-46](https://user-images.githubusercontent.com/2376968/170493896-5481f925-4769-4eda-811e-a9e6b3d5c676.png)

We should make a point of documenting the intricacies of these forms...